build/artifacts.props

```diff
@@ -44,6 +44,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.MiddlewareAnalysis" Category="ship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Mvc.Analyzers" Category="ship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Mvc.Api.Analyzers" Category="ship" />
+    <PackageArtifact Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" Category="ship" />
```

Comment:
FYI this package will be in the no-mans-land category of "ships as a package but not in the shared framework"

If you're doing this as a temporary solution for now, or if this is the correct marking then that's fine.

Reply:
This is what this marking means, tell me if it's right:

* Not in the shared framework
* Should ship to nuget.org

Reply:
OK, cool beans!